v1/artifact.go

```diff
@@ -25,7 +25,7 @@ type ArtifactVersion struct {
 	Tag      string            `json:",omitempty"`
 	Type     string            `json:"artifact_type"`
 	ID       string            `json:"id"`
-	Version  int               `json:"version"`
+	Version  string            `json:"version"`
```

Comment:
So the ArtifactVersion "Version" is passed in as a string but returned as an int?